test_pytest_rerunfailures.py

```diff
@@ -511,3 +511,36 @@ def test_only_rerun_flag(testdir, only_rerun_texts, should_rerun):
     assert_outcomes(
         result, passed=num_passed, failed=num_failed, rerun=num_reruns_actual
     )
+
+
+@pytest.mark.parametrize("condition, expected_reruns", [(1 == 1, 2), (1 == 2, 0)])
```

Comment:
Maybe `1==1` etc should be just `True` and `False`? Or maybe they should be strings? 🤔

Reply:
As we have a parametrization anyway here, we could add some more cases. Both mentioned of you seem fine to me. This also serves as kind of documentation after all.

Reply:
Added more tests.